### PR TITLE
draft spring zeebe sdk

### PR DESCRIPTION
**Pull request by christinaausley**
## Description

Related to https://github.com/camunda/product-hub/issues/2148.

## When should this change go live?

- [x] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [ ] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [ ] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [ ] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [x] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

File: sidebars.js [hunk 1]
```diff
@@ -776,6 +776,16 @@ module.exports = {
         "apis-tools/build-your-own-client",
       ],
     },
+    {
+      SDKs: [
+        {
+          "Spring Zeebe": [
```
Comment (christinaausley) [inline sidebars.js:782]: Is this the name we want to use? I've interchangeably seen Spring Boot Starter Camunda SDK.

Comment (megglos) [reply sidebars.js:782]: Yes

**Review by megglos — DISMISSED** [review]
LGTM! Thank you so much for picking this up and integrating it right away into the docs! 🙇

**Comment by megglos** [comment]
@ana-vinogradova-camunda can you take a look here too? @christinaausley we don't need to await @MaxTru feedback here imho, from my perspective we are ready to merge once @ana-vinogradova-camunda and @aleksander-dytko took a look.

**Review by ana-vinogradova-camunda — APPROVED** [review]
LGTM, thank you 🚀

**Review by megglos — DISMISSED** [review]
👍

**Review by felix-mueller — APPROVED** [review]

**Review by ana-vinogradova-camunda — APPROVED** [review]

**Review by megglos — APPROVED** [review]
Thanks @aleksander-dytko for the suggestions!

**Pull request merged**